app/javascript/stylesheets/application.scss

```diff
@@ -6,3 +6,6 @@
 
 //Sidebar
 @import "./layouts/sidebar";
+
+//Contact table
+@import "./components//contact_table.scss"
```

Comment:
We wouldn't need this anymore since we will be switching to tailwind classnames.

Reply:
Also, while writing `scss` partials like you have in this case, please prefix the filename with `_` 
like - `components/_contact-table.scss`.
While importing the file, you can simply write - `components/contact-table`.